src/Schema/Events/MyCollection.ts

```diff
@@ -26,8 +26,10 @@ export type Platform = "web" | "mobile"
  */
 export interface AddCollectedArtwork {
   action: ActionType.addCollectedArtwork
-  context_module: ContextModule.myCollectionHome
-  context_owner_type: OwnerType.myCollection
+  context_module:
+    | ContextModule.myCollectionHome
+    | ContextModule.myCollectionOnboarding
+  context_owner_type: OwnerType.myCollection | OwnerType.myCollectionOnboarding
```

Comment:
**issue:** I would suggest moving the event to `/Events/HomeFeedMyCollectionOnboarding.ts` since it's part of the homefeed onboarding. Keeping it here gives the impression it's part of My Collection screen

Reply:
We can of course rename it slightly

Reply:
Keeping one single name for the event (only having 2 different context_owner_type) makes the pipeline update much easier, if that's possible

Reply:
Oki sounds reasonable